shell/nuxt/components/nuxt-build-indicator.vue

```diff
@@ -0,0 +1,143 @@
+<template>
```

Comment:
Can we document/comment concisely somewhere what is this code from the `shell/nuxt` folder for, why do we need it and what are we planning to do?
Up to you either with a commit or an issue.